usaspending_api/etl/tests/integration/test_load_transactions_in_delta.py

```diff
@@ -1345,11 +1324,6 @@ def unexpected_paths_source_tables_only_test_core(self):
         # First, load the source tables to Delta
         _load_tables_to_delta(self.s3_data_bucket)
 
-        # 1. Test calling load_transactions_in_delta with etl-level of transaction_f[ab|pd]s before calling with
-        # etl-level of initial_run.
-        with raises(pyspark.sql.utils.AnalysisException, match=f"Table or view not found: int.{self.etl_level}"):
-            call_command("load_transactions_in_delta", "--etl-level", self.etl_level)
-
         # 2. Call load_transactions_in_delta with etl-level of initial_run first, but without first loading
```

Comment:
If we're removing `# 1.` here, can we update this to `# 1.` and the `# 3.` on line 1346 to `# 2.`?

Reply:
Updated